sklearn/tests/test_pipeline.py

```diff
@@ -827,7 +827,21 @@ def test_set_feature_union_steps():
     assert_equal(['mock__x5'], ft.get_feature_names())
 
 
-def test_set_feature_union_step_none():
+def test_set_feature_union_test_none():
```

Comment:
same comment here as in the other PR, might make sense to parametrize for None / 'drop'